docs/articles/documentation/using-testcafe/common-concepts/browser-support.md

```diff
@@ -88,6 +92,77 @@ You can search npm for plugins developed by the community. Their names begin wit
+
+* [Running Tests in Headless Mode](#running-tests-in-headless-mode)
+* [Running Tests in the Device Emulation Mode](#running-tests-in-the-device-emulation-mode)
+
+### Running Tests in Headless Mode
+
+TestCafe allows you to run tests in Google Chrome without any visible UI shell - in the [headless mode](https://developers.google.com/web/updates/2017/04/headless-chrome). Note that the [Take screenshot](../../test-api/actions/take-screenshot.html) and [Resize window](../../test-api/actions/resize-window.html) features are fully supported in headless mode.
+
+```sh
+testcafe "chrome:headless" tests/sample-fixture.js
+```
+
+You can specify a custom port for CDP:
+
+```sh
+testcafe "chrome:headless:cdpPort=9223" tests/sample-fixture.js
+```
+
+Note that headless mode is available for Mac and Linux in Chrome 59. You need a [Canary build](https://www.google.com/chrome/browser/canary.html) to enable headless mode on Windows. Specify a path to installation location, if you install a portable version of Chrome:
+
+```sh
+testcafe "chrome:path/to/chrome:headless" tests/sample-fixture.js
+```
+
+### Running Tests in the Device Emulation Mode
+
+You can run test in Chrome's built-in [device emulator](https://developers.google.com/web/tools/chrome-devtools/device-mode/):
+
+```sh
+testcafe "chrome:emulation:device=iphone 6" tests/sample-fixture.js
+testcafe "chrome:emulation:width=100;height=200;mobile=true;orientation=vertical;touch=true" tests/sample-fixture.js
+```
+
+You can combine both device emulation and headless modes:
+
+```sh
+testcafe "chrome:headless:emulation:device=iphone 6;cdpPort=9223" tests/sample-fixture.js
+```
+
+### Available CDP options
+
+Usage:
+
+```sh
+chrome[:pathToBrowser][:headless][:emulation][:arg1=value1;arg2=value2][ --cmdArgs]
+```
+
+Options:
+
+* **pathToBrowser** - specifies a path to a portable browser. The path to browser is not required, if it is installed in a system.
+* **headless** - Chrome runs in [headless mode](https://developers.google.com/web/updates/2017/04/headless-chrome).
+* **emulation** - allows you to run tests in Chrome [device emulator](https://developers.google.com/web/tools/chrome-devtools/device-mode/).
+* **chrome arguments** - options separated by `;`.  The following arguments are available:
+    * **device parameters**:
+        * **device=\<string\>** - the emulated device;
+        * **width=\<number\>** - the device screen width in pixels;
+        * **height=\<number\>** - the device screen height in pixels;
+        * **scaleFactor=\<number\>** - device scale factor value;
+        * **mobile=\<bool\>** - whether to emulate a mobile device;
+        * **orientation=\<vertical|horizontal\>** - the device orientation;
+        * **userAgent=\<string\>** - the user agent string;
+        * **touch=\<bool\>** - whether the touch support is enabled.
+    * **CDP arguments**:
+        * **cdpPort=\<int\>** - a free port used for the Chrome Debugging Protocol.
```

Comment:
**cdpPort=\<number\>** - a free port (0-65535) used for the Chrome Debugging Protocol.